### PR TITLE
Fix MM*DataPreProcessor issue with a more elegant way

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary

- Please see https://github.com/openvinotoolkit/training_extensions/issues/2743 for the history of this work.

### How to test
Existing tests can cover it.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Review by sungmanc — DISMISSED** [review]
LGTM

File: src/otx/core/model/entity/instance_segmentation.py [hunk 1]
```diff
@@ -131,7 +149,10 @@ def _customize_outputs(
                     canvas_size=output.img_shape,
                 ),
             )
-            output_masks = tv_tensors.Mask(output.pred_instances.masks, dtype=torch.bool)
+            output_masks = tv_tensors.Mask(
+                output.pred_instances.masks,
+                dtype=torch.bool,
+            )
```
Comment (jaegukhyun) [inline src/otx/core/model/entity/instance_segmentation.py:155]: Is this change come from ruff-formatter?

Comment (vinnamkim) [reply src/otx/core/model/entity/instance_segmentation.py:155]: No but ruff-formatter allowed it

**Review by jaegukhyun — DISMISSED** [review]
LGTM, is there issue ticket in MMEngine for this kind of bug?

**Comment by vinnamkim** [comment]
> LGTM, is there issue ticket in MMEngine for this kind of bug?

I don't know. It might be not a bug in their perspective. We may create a ticket in their repo.

**Review by jaegukhyun — DISMISSED** [review]

**Review by jaegukhyun — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Pull request merged**